### PR TITLE
[Snackbar] Removed unused variable from MDCSnackbarMessageView

### PR DESCRIPTION
**Pull request by trungducc**
#### Don't forget:
- [x] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [x] Sign the CLA bot. You can do this once the pull request is opened.

[Contributing](./contributing/README.md#pull-requests) has more information and tips for a great
pull request.

### CODE REVIEW

**Review by jverkoey — APPROVED** [review]

**Comment by jverkoey** [comment]
Thank you!

File: components/Snackbar/src/MDCSnackbarMessageView.m [hunk 1]
```diff
@@ -424,8 +410,6 @@ - (void)initializeMDCSnackbarMessageViewButtons:(MDCSnackbarMessage *)message {
     CGFloat buttonContentPadding =
```
Comment (yarneo) [inline components/Snackbar/src/MDCSnackbarMessageView.m]: This becomes unused too

File: components/Snackbar/src/MDCSnackbarMessageView.m [hunk 2]
```diff
@@ -424,8 +410,6 @@ - (void)initializeMDCSnackbarMessageViewButtons:(MDCSnackbarMessage *)message {
...
         MDCSnackbarMessage.usesLegacySnackbar ? kLegacyButtonPadding : kButtonPadding;
     CGSize buttonSize = [button sizeThatFits:CGSizeMake(CGFLOAT_MAX,CGFLOAT_MAX)];
```
Comment (yarneo) [inline components/Snackbar/src/MDCSnackbarMessageView.m]: This also becomes unused

**Review by yarneo — CHANGES_REQUESTED** [review]
Thanks so much for cleaning our code!! Due to the removal there were 2 other variables that became unused too. Please remove them also when you have the chance so the target will build successfully.

**Comment by trungducc** [comment]
Thanks for correcting me. I have fixed commit with suggestions in above comment. I tried to run `scripts/check_components components/Snackbar` to check component but I got stuck with `scripts/build_site.sh`. I wonder that there is any official document with steps for me to run `scripts/check_components` right way.

**Comment by yarneo** [comment]
thank you! @jverkoey is there a good way for contributors to test kokoro or if the build passes locally?

**Review by yarneo — APPROVED** [review]

**Comment by jverkoey** [comment]
Running `.kokoro` should run the local tests, though I expect it will require installing bazel.

**Comment by jverkoey** [comment]
Easiest way to run tests with minimal installation steps is to open MDCCatalog and run tests.

**Pull request merged**